src/main/java/com/bandwidth/sdk/numbers/models/NumbersApiError.java

```diff
@@ -0,0 +1,16 @@
+package com.bandwidth.sdk.numbers.models;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.immutables.value.Value;
+
+import java.util.List;
+
+@Value.Immutable
+@JsonSerialize(as = ImmutableNumbersApiError.class)
+@JsonDeserialize(as = ImmutableNumbersApiError.class)
+public abstract class NumbersApiError {
```

Comment:
I think the Numbers error's look different than our Messages api. We should probably just remove this (And NumbersApiFieldError)

Reply:
I think that's right. Based on the docs, all of the responses look like this

```
<BillingReportCreationResponse>
    <ResponseStatus>
        <ErrorCode>15501</ErrorCode>
        <Description>Invalid date range. Start date 2014-05-21 must be before end date 2013-05-29</Description>
    </ResponseStatus>
</BillingReportCreationResponse>
```

All have the same ResponseStatus fields, but the outer tag looks different for all of them